src/clj/game/core/runs.clj

```diff
@@ -370,10 +370,10 @@
   title-fn: a function taking a card map being accessed and returning a string to print as the card's title, e.g.,
       'an unseen card from R&D' for an R&D run.
   already-accessed: a set of cards already accessed from this zone or its root."
-
+  [state c-zone label amount select-fn title-fn already-accessed]
   (let [get-root-content (fn [state]
-                           (filter #(not (contains? already-accessed %)) (get-in @state [:corp :servers zone :content])))
-        server-name (central->name zone)
+                           (filter #(not (contains? already-accessed %)) (get-in @state [:corp :servers c-zone :content])))
+        server-name (central->name c-zone)
```

Comment:
I'd call this `card-zone`, unless that's also a fn... in which case `c-zone` is acceptable I guess.

Reply:
In this instance, it's just `:rd` or `:hq`, so I think `card-zone` works.